docs/react-native/integration/add-embrace-sdk.md

```diff
@@ -71,27 +71,9 @@ correctly.
 <Tabs groupId="platform" queryString="platform">
 <TabItem value="ios" label="iOS">
 
-Configuration for iOS is handled in code when initializing the SDK which we will cover in the next step. If you're on
-React Native version 0.60 and above you benefit from [Autolinking](https://github.com/react-native-community/cli/blob/dec33cb945be548a0d30c2ea073493e253239850/docs/autolinking.md#platform-ios)
-to set up the native module so you're good to go! Otherwise review the section below:
-
-### React Native Version < 0.60
```

Comment:
Also removed this to cleanup this section a bit, versions older than 0.60 are 5+ years at this point and no longer included on this list: https://reactnative.dev/versions